examples/08_data_block.rs

```diff
@@ -16,22 +16,19 @@ async fn main() -> Result<()> {
         .finish()
         .await?;
 
-    let message = client
-        .message()
+    let block = client
+        .block()
         .with_tag("Hello")
         .with_data("Tangle".as_bytes().to_vec())
         .finish()
         .await?;
 
-    println!(
-        "Message sent https://explorer.iota.org/devnet/message/{}\n",
-        message.id()
-    );
+    println!("Block sent https://explorer.iota.org/devnet/block/{}\n", block.id());
```

Comment:
Is this URL correct ?

Reply:
Currently it doesn't work anyways, we need to update when we have a public explorer for stardust